Example/Backpack/Images.xcassets/tab_explore_with_dot_dm.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "ic_explore_dot_deselected_darkmode.pdf",
+      "idiom" : "universal",
```

Comment:
I think by only specifying 1x PDF it automatically creates 2x and 3x at build? Not sure though

Reply:
Yes that's what it does and because of that I wouldn't expect to see `2x` and `3x` values in the asset catalog 🤔 Typically assets like this only have a single entry in `images`

Reply:
Ah right - I assumed Xcode would just add them back in but apparently it doesn't